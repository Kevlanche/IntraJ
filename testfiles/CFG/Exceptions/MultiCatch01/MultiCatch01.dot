digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"; "ClassDecl[77]"; "ClassDecl[84]"}
{ rank = same; "Modifiers[6]"; "Opt[8]"; "List[9]"; "List[10]"; "Modifiers[78]"; "Opt[80]"; "List[82]"; "List[83]"; "Modifiers[85]"; "Opt[87]"; "List[89]"; "List[90]"}
{ rank = same; "List[7]"; "MethodDecl[11]"; "List[79]"; "TypeAccess[81]"; "List[86]"; "TypeAccess[88]"}
{ rank = same; "Entry[12]"; "Modifiers[13]"; "PrimitiveTypeAccess[15]"; "List[16]"; "List[17]"; "Opt[18]"; "Exit[19]"}
{ rank = same; "List[14]"; "Block[19]"}
{ rank = same; "List[20]"}
{ rank = same; "TryStmt[21]"; "VarDeclStmt[69]"}
{ rank = same; "Block[22]"; "List[49]"; "Opt[67]"; "UncheckedExceptions[68]"; "Modifiers[70]"; "PrimitiveTypeAccess[72]"; "List[73]"}
{ rank = same; "List[23]"; "MultiCatch[50]"; "List[71]"; "VariableDeclarator[74]"}
{ rank = same; "IfStmt[24]"; "CatchParameterDeclaration[51]"; "Block[57]"; "List[75]"; "Opt[76]"}
{ rank = same; "BooleanLiteral[25]"; "Block[26]"; "Opt[37]"; "Modifiers[52]"; "List[54]"; "List[58]"}
{ rank = same; "List[27]"; "Block[38]"; "List[53]"; "TypeAccess[55]"; "TypeAccess[56]"; "VarDeclStmt[59]"}
{ rank = same; "ThrowStmt[28]"; "List[39]"; "Modifiers[60]"; "PrimitiveTypeAccess[62]"; "List[63]"}
{ rank = same; "ClassInstanceExpr[30]"; "UncheckedExceptions[29]"; "List[36]"; "ThrowStmt[40]"; "List[61]"; "VariableDeclarator[64]"}
{ rank = same; "TypeAccess[31]"; "List[32]"; "Opt[33]"; "List[34]"; "List[35]"; "ClassInstanceExpr[42]"; "UncheckedExceptions[41]"; "List[48]"; "List[65]"; "Opt[66]"}
{ rank = same; "TypeAccess[43]"; "List[44]"; "Opt[45]"; "List[46]"; "List[47]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < MultiCatch01 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[8]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[9]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[11]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[12]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[13]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[14]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[15]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[16]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[18]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[19]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[20]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryStmt[21]" [label="TryStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[22]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[23]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IfStmt[24]" [label="IfStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BooleanLiteral[25]" [label="BooleanLiteral\n < true >", fillcolor=white   style=filled  ]
"Block[26]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[27]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ThrowStmt[28]" [label="ThrowStmt\n ", fillcolor=white   style=filled  ]
"ClassInstanceExpr[30]" [label="ClassInstanceExpr\n <TestException1>", fillcolor=white   style=filled  ]
"TypeAccess[31]" [label="TypeAccess\n < TestException1 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[33]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[35]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[29]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[37]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[38]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[39]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ThrowStmt[40]" [label="ThrowStmt\n ", fillcolor=white   style=filled  ]
"ClassInstanceExpr[42]" [label="ClassInstanceExpr\n <TestException2>", fillcolor=white   style=filled  ]
"TypeAccess[43]" [label="TypeAccess\n < TestException2 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[44]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[45]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[46]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[47]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[41]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[48]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[49]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MultiCatch[50]" [label="MultiCatch\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CatchParameterDeclaration[51]" [label="CatchParameterDeclaration\n ", fillcolor=white   style=filled  ]
"Modifiers[52]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[53]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[54]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[55]" [label="TypeAccess\n < TestException1 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[56]" [label="TypeAccess\n < TestException2 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[57]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[58]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[59]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[60]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[61]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[62]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[63]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[64]" [label="VariableDeclarator\n < int a >", fillcolor=white   style=filled  ]
"List[65]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[66]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[67]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[68]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[69]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[70]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[71]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[72]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[73]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[74]" [label="VariableDeclarator\n < int b >", fillcolor=white   style=filled  ]
"List[75]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[76]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[19]" [label="Exit\n ", fillcolor=white   style=filled  ]
"ClassDecl[77]" [label="ClassDecl\n < TestException1 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[78]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[79]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[80]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[81]" [label="TypeAccess\n < Exception >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[82]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[83]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[84]" [label="ClassDecl\n < TestException2 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[85]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[86]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[87]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[88]" [label="TypeAccess\n < Exception >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[89]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[90]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[8]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[9]" [style=dashed, color=gray]
"MethodDecl[11]" -> "Entry[12]" [style=dashed, color=gray]
"Modifiers[13]" -> "List[14]" [style=dashed, color=gray]
"MethodDecl[11]" -> "Modifiers[13]" [style=dashed, color=gray]
"MethodDecl[11]" -> "PrimitiveTypeAccess[15]" [style=dashed, color=gray]
"MethodDecl[11]" -> "List[16]" [style=dashed, color=gray]
"MethodDecl[11]" -> "List[17]" [style=dashed, color=gray]
"IfStmt[24]" -> "BooleanLiteral[25]" [style=dashed, color=gray]
"ClassInstanceExpr[30]" -> "TypeAccess[31]" [style=dashed, color=gray]
"ClassInstanceExpr[30]" -> "List[32]" [style=dashed, color=gray]
"ClassInstanceExpr[30]" -> "Opt[33]" [style=dashed, color=gray]
"ClassInstanceExpr[30]" -> "List[34]" [style=dashed, color=gray]
"ClassInstanceExpr[30]" -> "List[35]" [style=dashed, color=gray]
"ThrowStmt[28]" -> "ClassInstanceExpr[30]" [style=dashed, color=gray]
"ThrowStmt[28]" -> "UncheckedExceptions[29]" [style=dashed, color=gray]
"ThrowStmt[28]" -> "List[36]" [style=dashed, color=gray]
"List[27]" -> "ThrowStmt[28]" [style=dashed, color=gray]
"Block[26]" -> "List[27]" [style=dashed, color=gray]
"IfStmt[24]" -> "Block[26]" [style=dashed, color=gray]
"ClassInstanceExpr[42]" -> "TypeAccess[43]" [style=dashed, color=gray]
"ClassInstanceExpr[42]" -> "List[44]" [style=dashed, color=gray]
"ClassInstanceExpr[42]" -> "Opt[45]" [style=dashed, color=gray]
"ClassInstanceExpr[42]" -> "List[46]" [style=dashed, color=gray]
"ClassInstanceExpr[42]" -> "List[47]" [style=dashed, color=gray]
"ThrowStmt[40]" -> "ClassInstanceExpr[42]" [style=dashed, color=gray]
"ThrowStmt[40]" -> "UncheckedExceptions[41]" [style=dashed, color=gray]
"ThrowStmt[40]" -> "List[48]" [style=dashed, color=gray]
"List[39]" -> "ThrowStmt[40]" [style=dashed, color=gray]
"Block[38]" -> "List[39]" [style=dashed, color=gray]
"Opt[37]" -> "Block[38]" [style=dashed, color=gray]
"IfStmt[24]" -> "Opt[37]" [style=dashed, color=gray]
"List[23]" -> "IfStmt[24]" [style=dashed, color=gray]
"Block[22]" -> "List[23]" [style=dashed, color=gray]
"TryStmt[21]" -> "Block[22]" [style=dashed, color=gray]
"Modifiers[52]" -> "List[53]" [style=dashed, color=gray]
"CatchParameterDeclaration[51]" -> "Modifiers[52]" [style=dashed, color=gray]
"List[54]" -> "TypeAccess[55]" [style=dashed, color=gray]
"List[54]" -> "TypeAccess[56]" [style=dashed, color=gray]
"CatchParameterDeclaration[51]" -> "List[54]" [style=dashed, color=gray]
"MultiCatch[50]" -> "CatchParameterDeclaration[51]" [style=dashed, color=gray]
"Modifiers[60]" -> "List[61]" [style=dashed, color=gray]
"VarDeclStmt[59]" -> "Modifiers[60]" [style=dashed, color=gray]
"VarDeclStmt[59]" -> "PrimitiveTypeAccess[62]" [style=dashed, color=gray]
"VariableDeclarator[64]" -> "List[65]" [style=dashed, color=gray]
"VariableDeclarator[64]" -> "Opt[66]" [style=dashed, color=gray]
"List[63]" -> "VariableDeclarator[64]" [style=dashed, color=gray]
"VarDeclStmt[59]" -> "List[63]" [style=dashed, color=gray]
"List[58]" -> "VarDeclStmt[59]" [style=dashed, color=gray]
"Block[57]" -> "List[58]" [style=dashed, color=gray]
"MultiCatch[50]" -> "Block[57]" [style=dashed, color=gray]
"List[49]" -> "MultiCatch[50]" [style=dashed, color=gray]
"TryStmt[21]" -> "List[49]" [style=dashed, color=gray]
"TryStmt[21]" -> "Opt[67]" [style=dashed, color=gray]
"TryStmt[21]" -> "UncheckedExceptions[68]" [style=dashed, color=gray]
"List[20]" -> "TryStmt[21]" [style=dashed, color=gray]
"Modifiers[70]" -> "List[71]" [style=dashed, color=gray]
"VarDeclStmt[69]" -> "Modifiers[70]" [style=dashed, color=gray]
"VarDeclStmt[69]" -> "PrimitiveTypeAccess[72]" [style=dashed, color=gray]
"VariableDeclarator[74]" -> "List[75]" [style=dashed, color=gray]
"VariableDeclarator[74]" -> "Opt[76]" [style=dashed, color=gray]
"List[73]" -> "VariableDeclarator[74]" [style=dashed, color=gray]
"VarDeclStmt[69]" -> "List[73]" [style=dashed, color=gray]
"List[20]" -> "VarDeclStmt[69]" [style=dashed, color=gray]
"Block[19]" -> "List[20]" [style=dashed, color=gray]
"Opt[18]" -> "Block[19]" [style=dashed, color=gray]
"MethodDecl[11]" -> "Opt[18]" [style=dashed, color=gray]
"MethodDecl[11]" -> "Exit[19]" [style=dashed, color=gray]
"List[10]" -> "MethodDecl[11]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"Modifiers[78]" -> "List[79]" [style=dashed, color=gray]
"ClassDecl[77]" -> "Modifiers[78]" [style=dashed, color=gray]
"Opt[80]" -> "TypeAccess[81]" [style=dashed, color=gray]
"ClassDecl[77]" -> "Opt[80]" [style=dashed, color=gray]
"ClassDecl[77]" -> "List[82]" [style=dashed, color=gray]
"ClassDecl[77]" -> "List[83]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[77]" [style=dashed, color=gray]
"Modifiers[85]" -> "List[86]" [style=dashed, color=gray]
"ClassDecl[84]" -> "Modifiers[85]" [style=dashed, color=gray]
"Opt[87]" -> "TypeAccess[88]" [style=dashed, color=gray]
"ClassDecl[84]" -> "Opt[87]" [style=dashed, color=gray]
"ClassDecl[84]" -> "List[89]" [style=dashed, color=gray]
"ClassDecl[84]" -> "List[90]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[84]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[19]" -> "UncheckedExceptions[29]" [color=red, constraint=false]
"Exit[19]" -> "UncheckedExceptions[41]" [color=red, constraint=false]
"Exit[19]" -> "VariableDeclarator[74]" [color=red, constraint=false]
"VariableDeclarator[74]" -> "Exit[19]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[74]" -> "VariableDeclarator[64]" [color=red, constraint=false]
"VariableDeclarator[64]" -> "VariableDeclarator[74]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[64]" -> "CatchParameterDeclaration[51]" [color=red, constraint=false]
"CatchParameterDeclaration[51]" -> "VariableDeclarator[64]" [color=blue, constraint=false, xlabel=" "] 
"CatchParameterDeclaration[51]" -> "UncheckedExceptions[29]" [color=red, constraint=false]
"CatchParameterDeclaration[51]" -> "UncheckedExceptions[41]" [color=red, constraint=false]
"UncheckedExceptions[29]" -> "CatchParameterDeclaration[51]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[29]" -> "Exit[19]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[29]" -> "ClassInstanceExpr[30]" [color=red, constraint=false]
"ClassInstanceExpr[30]" -> "UncheckedExceptions[29]" [color=blue, constraint=false, xlabel=" "] 
"ClassInstanceExpr[30]" -> "ThrowStmt[28]" [color=red, constraint=false]
"ThrowStmt[28]" -> "ClassInstanceExpr[30]" [color=blue, constraint=false, xlabel=" "] 
"ThrowStmt[28]" -> "BooleanLiteral[25]" [color=red, constraint=false]
"UncheckedExceptions[41]" -> "CatchParameterDeclaration[51]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[41]" -> "Exit[19]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[41]" -> "ClassInstanceExpr[42]" [color=red, constraint=false]
"ClassInstanceExpr[42]" -> "UncheckedExceptions[41]" [color=blue, constraint=false, xlabel=" "] 
"ClassInstanceExpr[42]" -> "ThrowStmt[40]" [color=red, constraint=false]
"ThrowStmt[40]" -> "ClassInstanceExpr[42]" [color=blue, constraint=false, xlabel=" "] 
"ThrowStmt[40]" -> "BooleanLiteral[25]" [color=red, constraint=false]
"BooleanLiteral[25]" -> "ThrowStmt[28]" [color=blue, constraint=false, xlabel="T "] 
"BooleanLiteral[25]" -> "ThrowStmt[40]" [color=blue, constraint=false, xlabel="F "] 
"BooleanLiteral[25]" -> "Entry[12]" [color=red, constraint=false]
"Entry[12]" -> "BooleanLiteral[25]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"ClassDecl[5]" -> "ClassDecl[77]" -> "ClassDecl[84]" [style=invis]
"Modifiers[6]" -> "Opt[8]" -> "List[9]" -> "List[10]" [style=invis]
"Entry[12]" -> "Modifiers[13]" -> "PrimitiveTypeAccess[15]" -> "List[16]" -> "List[17]" -> "Opt[18]" -> "Exit[19]" [style=invis]
"TryStmt[21]" -> "VarDeclStmt[69]" [style=invis]
"Block[22]" -> "List[49]" -> "Opt[67]" [style=invis]
"BooleanLiteral[25]" -> "Block[26]" -> "Opt[37]" [style=invis]
"TypeAccess[31]" -> "List[32]" -> "Opt[33]" [style=invis]
"TypeAccess[43]" -> "List[44]" -> "Opt[45]" [style=invis]
"CatchParameterDeclaration[51]" -> "Block[57]" [style=invis]
"Modifiers[52]" -> "List[54]" [style=invis]
"TypeAccess[55]" -> "TypeAccess[56]" [style=invis]
"Modifiers[60]" -> "PrimitiveTypeAccess[62]" -> "List[63]" [style=invis]
"List[65]" -> "Opt[66]" [style=invis]
"Modifiers[70]" -> "PrimitiveTypeAccess[72]" -> "List[73]" [style=invis]
"List[75]" -> "Opt[76]" [style=invis]
"Modifiers[78]" -> "Opt[80]" -> "List[82]" -> "List[83]" [style=invis]
"Modifiers[85]" -> "Opt[87]" -> "List[89]" -> "List[90]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
92", fillcolor=1]
Edges [label="Total Nr. Edges
119", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
12", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
14", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}