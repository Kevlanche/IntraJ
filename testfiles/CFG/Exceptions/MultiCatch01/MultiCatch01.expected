NodeName: Entry[12] Successors: [BooleanLiteral[25]]
NodeName: BooleanLiteral[25] Successors: [ThrowStmt[28], ThrowStmt[40]]
NodeName: ThrowStmt[28] Successors: [ClassInstanceExpr[30]]
NodeName: ClassInstanceExpr[30] Successors: [UncheckedExceptions[29]]
NodeName: UncheckedExceptions[29] Successors: [CatchParameterDeclaration[51], Exit[19]]
NodeName: CatchParameterDeclaration[51] Successors: [VariableDeclarator[64]]
NodeName: VariableDeclarator[64] Successors: [VariableDeclarator[74]]
NodeName: VariableDeclarator[74] Successors: [Exit[19]]
NodeName: Exit[19] Successors: []
NodeName: ThrowStmt[40] Successors: [ClassInstanceExpr[42]]
NodeName: ClassInstanceExpr[42] Successors: [UncheckedExceptions[41]]
NodeName: UncheckedExceptions[41] Successors: [CatchParameterDeclaration[51], Exit[19]]

NodeName: Exit[19] Predecessor: [UncheckedExceptions[29], UncheckedExceptions[41], VariableDeclarator[74]]
NodeName: UncheckedExceptions[29] Predecessor: [ClassInstanceExpr[30]]
NodeName: ClassInstanceExpr[30] Predecessor: [ThrowStmt[28]]
NodeName: ThrowStmt[28] Predecessor: [BooleanLiteral[25]]
NodeName: BooleanLiteral[25] Predecessor: [Entry[12]]
NodeName: Entry[12] Predecessor: []
NodeName: UncheckedExceptions[41] Predecessor: [ClassInstanceExpr[42]]
NodeName: ClassInstanceExpr[42] Predecessor: [ThrowStmt[40]]
NodeName: ThrowStmt[40] Predecessor: [BooleanLiteral[25]]
NodeName: VariableDeclarator[74] Predecessor: [VariableDeclarator[64]]
NodeName: VariableDeclarator[64] Predecessor: [CatchParameterDeclaration[51]]
NodeName: CatchParameterDeclaration[51] Predecessor: [UncheckedExceptions[29], UncheckedExceptions[41]]
